grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx

```diff
@@ -0,0 +1,265 @@
+import React, { FC, SyntheticEvent, useRef, useState } from 'react';
+
+import { Icon, LoadingPlaceholder } from '@grafana/ui';
+import cn from 'classnames/bind';
+
+import Tag from 'components/Tag/Tag';
+import Text from 'components/Text/Text';
+import { WithContextMenu } from 'components/WithContextMenu/WithContextMenu';
+import { WithPermissionControl } from 'containers/WithPermissionControl/WithPermissionControl';
+import { Alert, AlertAction, IncidentStatus } from 'models/alertgroup/alertgroup.types';
+import styles from 'pages/incidents/parts/IncidentDropdown.module.scss';
+import { UserActions } from 'utils/authorization';
+
+import { SilenceSelect } from './SilenceSelect';
+
+const cx = cn.bind(styles);
+
+const getIncidentTagColor = (alert: Alert) => {
+  if (alert.status === IncidentStatus.Resolved) {
+    return '#299C46';
```

Comment:
it's better to define such color values in `vars.css`

Reply:
Not sure I understood your point, this is pure JS, how is this related to `vars.css`? We do not import variables from there into JS, or do we?

I have a separate PR where I moved those to `consts.ts` - if you approve that one first I can get the changes here too.

Reply:
my point is to save all color values in `vars.css` and use them as `var(--some-color)`, this approach supports theming

Reply:
Done